packages/flutter_tools/lib/src/commands/analyze_base.dart

```diff
@@ -85,6 +85,27 @@ abstract class AnalyzeBase {
   }
 
   bool get isBenchmarking => argResults['benchmark'] as bool;
+
+  String generateDartDocMessage(int undocumentedMembers) {
```

Comment:
You could make this easier to test by making it a static or top level method, since it doesn't depend on anything in AnalyzeBase.